### PR TITLE
Add `AssetImage.inCarousel` field and input

### PR DESCRIPTION
**Pull request by B77Mills**

### CODE REVIEW

**Comment by B77Mills** [comment]
[Article slideshows have not transferred over](https://trello.com/c/Khw0uYuk/129-article-slideshows-have-not-transferred-over)

File: services/graphql-server/src/graphql/definitions/platform/asset.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ type AssetImage {
   caption: String @projection
   credit: String @projection
   cropDimensions: AssetImageCrop @projection
+  inCarousel: Boolean @projection
```
Comment (zarathustra323) [inline services/graphql-server/src/graphql/definitions/platform/asset.js:35]: Should this also be _is_ (instead of _in_) to match L106 below?

**Review by zarathustra323 — DISMISSED** [review]

File: services/graphql-server/src/graphql/definitions/platform/asset.js (hunk 1 above)
Comment (B77Mills) [reply services/graphql-server/src/graphql/definitions/platform/asset.js:35]: @zarathustra323 I just matched what is currently being saved on assets.  If we would prefer it to be `is`.  I can update it within management to the new field name and convert any data that has it already set.  

Thoughts?

Comment (zarathustra323) [reply services/graphql-server/src/graphql/definitions/platform/asset.js:35]: Then I'd leave it as `in` on the `Type` but change the `is` on the `Input` to `in` -- just so things are consistent

**Review by solocommand — APPROVED** [review]

**Pull request merged**